.goreleaser.yaml

```diff
@@ -38,10 +38,14 @@ builds:
     tags:
       - ledger
       - multiplexer
+    flags:
+      - -trimpath
```

Comment:
-trimpath
	•	What it does: Removes all file system paths from the compiled binary.
	•	Why it exists: By default, Go embeds absolute paths to your source files (e.g., /home/runner/work/myrepo/...) into the binary. This can:
	•	Make builds less reproducible (different machines → different paths).
	•	Leak environment details you don’t want in distributed binaries.
	•	Effect: Cleaner, more reproducible builds with slightly smaller binaries. No effect on runtime behavior.